src/Console/ConfigurationResolver.php

```diff
@@ -350,6 +350,10 @@ private function parseRules()
         $rules = array();
 
         foreach (array_map('trim', explode(',', $this->options['rules'])) as $rule) {
+            if ('' === $rule) {
+                throw new InvalidConfigurationException('Empty rule name is not allowed.');
+            }
+
             if ('-' === $rule[0]) {
```

Comment:
then why `-` and not `^` or any other char?
In the end, "invalid char in rule name" is just a special case of "name of non-existing rule"